docs/SETTINGS.rst

```diff
@@ -103,6 +103,7 @@ Most of the parameters are optional, but you have to specify one of the **host**
 -  **consistency**: (optional) Select consul consistency mode. Possible values are ``default``, ``consistent``, or ``stale`` (more details in `consul API reference <https://www.consul.io/api/features/consistency.html/>`__)
 -  **checks**: (optional) list of Consul health checks used for the session. By default an empty list is used.
 -  **register\_service**: (optional) whether or not to register a service with the name defined by the scope parameter and the tag master, replica or standby-leader depending on the node's role. Defaults to **false**.
+-  **service\_tags**: (optional) additional static tags to add to the Consul service apart from the role (``master``/``replica``/``standby-leader``).  By default an empty list is used.
```

Comment:
IIRC if there is more than one tag it will become impossible to rely on DNS to find the primary/replicas.

Reply:
I don't think so actually - at least it works fine for me for other services. Also note that this is optional, and defaulting to an empty list,